Libplanet/Store/BaseStore.cs

```diff
@@ -105,7 +105,7 @@ public Block<T> GetBlock<T>(HashDigest<SHA256> blockHash)
         /// <inheritdoc/>
         public long? GetBlockIndex(HashDigest<SHA256> blockHash)
         {
-            return GetRawBlock(blockHash)?.Index;
+            return GetBlockDigest(blockHash)?.Header.Index;
```

Comment:
Is it okay that we omit `?` on `.Header`?

```suggestion
            return GetBlockDigest(blockHash)?.Header?.Index;
```

Reply:
Yes, because `Header` always exists if `GetBlockDigest(blockHash)` isn't null.